app/assets/stylesheets/administrate/components/_cells.scss

```diff
@@ -26,10 +26,9 @@
 }
 
 .cell-label__sort-indicator {
-  display: inline-block;
   overflow: hidden;
-  position: absolute;
```

Comment:
Now that we are no longer using absolute positioning, can the `position: relative;` go away on Line 2?

Reply:
I think you're correct. I'll update it.